tests/src/test/java/me/aurium/beetle/branch/tests/AbstractTests.java

```diff
@@ -0,0 +1,23 @@
+package me.aurium.beetle.branch.tests;
+
+import me.aurium.beetle.branch.adapter.ContextAdapterFactory;
+import me.aurium.beetle.branch.adapter.DelegatingContextAdapterFactory;
+import me.aurium.beetle.branch.tests.mockups.MockupContextSource;
+import org.junit.platform.commons.logging.Logger;
+import org.junit.platform.commons.logging.LoggerFactory;
+
+public abstract class AbstractTests {
+
+    protected final Logger logger = LoggerFactory.getLogger(BranchingNodeTests.class);
+
+    protected final ContextAdapterFactory<String> adapter = new DelegatingContextAdapterFactory<>(new MockupContextSource(logger));
+
+    protected final static String SENDER = "Tester";
+    protected final static String ALIAS = "TestCommand";
+
+    //@A248 is there a better way to do this with junit5? i'm not used to it. Is ther also a way to make the logger messages less ugly? They look shitty compared to juni4
+    protected void logTestStarting(String name) {
+        logger.info(() -> "(BRANCH) Beginning test: " + name);
```

Comment:
I don't know why you're using `org.junit.platform.commons.logging` - that's internal API from junit-platform-commons. Your IDE should warn you such package is encapsulated. Maybe it's because you need to add module-info to Branch.

If you use slf4j, to format its log messages you have to configure the slf4j implementation. If that's slf4j-simple, make a simplelogger.properties and follow the slf4j-simple docs.

Reply:
Hmm, interesting. I tried slf4j with junit5 once and it didn't work which is why i defaulted to looking for a platform specific logger, but that makes much more sense

Reply:
It works fine. What "didn't work" about it?

Reply:
What didn't work was nothing was logged.